dockerfiles/idp_prod.Dockerfile

```diff
@@ -81,17 +149,20 @@ RUN ln -snf /usr/share/zoneinfo/$TZ /etc/localtime && echo $TZ > /etc/timezone
 # Create the working directory
 WORKDIR $RAILS_ROOT
 
+# bundle install (most of the work is already done in the builder image)
 COPY .ruby-version $RAILS_ROOT/.ruby-version
 COPY Gemfile $RAILS_ROOT/Gemfile
 COPY Gemfile.lock $RAILS_ROOT/Gemfile.lock
-
+COPY --from=builder $RAILS_ROOT/bin $RAILS_ROOT/bin
+COPY --from=builder $RAILS_ROOT/vendor $RAILS_ROOT/vendor
 RUN bundle config build.nokogiri --use-system-libraries
 RUN bundle config set --local deployment 'true'
 RUN bundle config set --local path $BUNDLE_PATH
 RUN bundle config set --local without 'deploy development doc test'
 RUN bundle install --jobs $(nproc)
 RUN bundle binstubs --all
 
+# yarn install
 COPY package.json $RAILS_ROOT/package.json
 COPY yarn.lock $RAILS_ROOT/yarn.lock
 RUN yarn install --production=true --frozen-lockfile --cache-folder .yarn-cache
```

Comment:
```suggestion
```

Reply:
Don't you want a yarn install?  This command adds 408MB to the image, so it's doing something.

Reply:
Yeah, probably a bunch of node_modules which we could copy to avoid installing twice.

Reply:
Though if we copy the compiled assets, we don't need to do that either.